### PR TITLE
[YarpSensorBridge] fix check for NaN

### PR DESCRIPTION
**Pull request by prashanthr05**
Fixes https://github.com/dic-iit/bipedal-locomotion-framework/issues/188

### CODE REVIEW

File: src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h [hunk 1]
```diff
@@ -143,6 +143,24 @@ struct YarpSensorBridge::Impl
...
+                        std::string_view logPrefix,
+                        const std::string& sensorName)
+    {
+        for (auto& val : vec)
+        {
+            if (std::isnan(val))
+            {
+                std::cerr << logPrefix << " NAN values read from " << sensorName << " , use previous measurement" << std::endl;
+	        return true;
+	    }
```
Comment (S-Dafarra) [inline src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h:160]: The indentation seems a little off

**Review by S-Dafarra — APPROVED** [review]
Just check the indentation, then it is ok for me

File: src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h (hunk 1 above)
Comment (prashanthr05) [reply src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h:160]: parallax error :D

Comment (prashanthr05) [reply src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h]: Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/191/commits/bad48a1bb5338489ab4071153574cf98463494b4

**Comment by S-Dafarra** [comment]
Awesome, for me feel free to merge as soon as CI passes.

**Comment by prashanthr05** [comment]
Merging, thanks!

**Pull request merged**